python/lsst/display/firefly/firefly.py

```diff
@@ -207,6 +208,19 @@ def _dot(self, symb, c, r, size, ctype, fontFamily="helvetica", textAngle=None):
     """
         self._uploadTextData(ds9Regions.dot(symb, c, r, size, ctype, fontFamily, textAngle))
 
+    def _overlayCatalog(self, sourceCat, **kwargs):
+        """ Upload a catalog and overlay on image display """
+        table = sourceCat.asAstropy()
+        table['ra'] = table['coord_ra'].to('deg')
+        table['dec'] = table['coord_dec'].to('deg')
+        with tempfile.NamedTemporaryFile(delete=False,
+                                         suffix='.csv') as fd:
+            table.write(fd.name, format='csv')
+
+        tbl_val = _fireflyClient.upload_file(fd.name)
+        os.remove(fd.name)
+        _fireflyClient.show_table(tbl_val)
```

Comment:
We might need to pass meta data eventually.  We would need to change firefly_client to take is as a parameter